auth_backends/pipeline.py

```diff
@@ -2,9 +2,12 @@
 
 For more info visit http://python-social-auth.readthedocs.org/en/latest/pipeline.html.
```

Comment:
I noticed that this link is broken, and should be updated to:
- https://python-social-auth.readthedocs.io/en/latest/pipeline.html